src/Strategies/DNSRecord.php

```diff
@@ -16,6 +18,23 @@ class DNSRecord implements StrategyInterface
      */
     public function verify(string $url, DomainVerifiableInterface $domainVerifiable)
     {
+        $dns = new Dns($url);
+        $txtRecords = $dns->getRecords('TXT');
+        $verificationName = config('domain_verifier.verification_name');
+        $txtRecords = explode(PHP_EOL, $txtRecords);
+        $domainToken = array();
+
+        foreach ($txtRecords as $item) {
```

Comment:
Should move this for into method `getToken($dnsRecords)`. It will return the given token in the dns records.